images.yml

```diff
@@ -54,17 +46,11 @@ config:
       description: "OpenJDK 11"
       major: 11
       fish-pepper:
-        # Ignore for for Alpine and JBoss as there are no Java 11 packages
+        # Ignore for for Alpine as there are no Java 11 packages
         ignore-for:
            - [ "alpine" ]
```

Comment:
I think the alpine ignore can be dropped too right?

Reply:
Just realized this is an old branch, a rebase would have the new stuff.

Reply:
thanks for the heads up, just rebased to master.